pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -230,18 +239,22 @@ func newGCECloud(config io.Reader) (*GCECloud, error) {
 		if cfg.Global.Multizone {
 			managedZones = nil // Use all zones in region
 		}
+
+		kmsConfig.Location = cfg.Global.Location
+		kmsConfig.KeyRing = cfg.Global.KeyRing
+		kmsConfig.CryptoKey = cfg.Global.CryptoKey
 	}
 
 	return CreateGCECloud(apiEndpoint, projectID, region, zone, managedZones, networkURL, subnetworkURL,
-		nodeTags, nodeInstancePrefix, tokenSource, true /* useMetadataServer */)
+		nodeTags, nodeInstancePrefix, tokenSource, true /* useMetadataServer */, kmsConfig)
 }
 
 // Creates a GCECloud object using the specified parameters.
 // If no networkUrl is specified, loads networkName via rest call.
 // If no tokenSource is specified, uses oauth2.DefaultTokenSource.
 // If managedZones is nil / empty all zones in the region will be managed.
 func CreateGCECloud(apiEndpoint, projectID, region, zone string, managedZones []string, networkURL, subnetworkURL string, nodeTags []string,
-	nodeInstancePrefix string, tokenSource oauth2.TokenSource, useMetadataServer bool) (*GCECloud, error) {
```

Comment:
blargh, this should be a struct with a list of fields, not a bunch of parameters. Not asking for a change, just complaining.